### DISCUSSION TITLE
New Features

### DISCUSSION THREAD

**Discussion by rcmsoftwaredevelopment** [serenity-is/Serenity#7023] [General]
Hey all

Thanks for all of the hard work in getting the framework to this point.  I'm just wondering what new features or samples are coming to the StartSharp framework over the next few months - Just purchased a new subscription.  I know maintenance and keeping it up todate is a full time job.

Best regards, Ray

**Comment by VictorTomaili** [comment]
Hello Ray,

We don't have a fixed schedule or a public backlog for the upcoming features. However, we are open to suggestions. We will evaluate the requests and decide whether to implement them based on their feasibility and usefulness.

**Comment by volkanceylan** [comment]
Just have a look at change log for last year: https://github.com/serenity-is/Serenity/blob/master/CHANGELOG.md and that should give you an idea.
